src/blocks/container/edit.js

```diff
@@ -40,7 +40,8 @@ const UAGBContainer = ( props ) => {
 		clientId,
 		setAttributes,
 		name,
-		deviceType
+		deviceType,
+		context
```

Comment:
Always add trailing comma.

```suggestion
		context,
```

Reply:
This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.